docs/ci-release.md

```diff
@@ -3,148 +3,184 @@
      - `stacks-blockchain:<x.x.x.x.x-debian>`
-5. A tagged release on the default branch will produce 2 versions of the docker image (along with all binary archives):
-   - An Alpine image for several architectures tagged with:
+5. A tagged release on the default branch will produce:
+   - Github Release of the specified tag with:
+     - Binary archives for several architectures
+   - Docker Alpine image for several architectures tagged with:
      - `stacks-blockchain:<x.x.x.x.x>`
      - `stacks-blockchain:<latest>`
-   - An Debian image for several architectures tagged with:
+   - Docker Debian image for several architectures tagged with:
      - `stacks-blockchain:<x.x.x.x.x-debian>`
      - `stacks-blockchain:<latest-debian>`
 
-## Release workflow:
-
-1. Create a feature branch: `feat/112-fix-something`
-2. PR `feat/112-fix-something` to the `develop` branch
-   1. CI Workflow is automatically triggered, resulting in a pushed docker image tagged with the **branch name** and **PR number**
-3. PR `develop` to the default branch
-   1. CI Workflow is automatically triggered, resulting in a pushed docker image tagged with the **branch name** and **PR number**
+## Release workflow
+
+1. Create a feature branch: `feat/fix-something`
+2. PR `feat/fix-something` to the `develop` branch where the PR is numbered `112`
+   1. Docker image tagged with the **branch name** and **PR number**
+   - ex:
+     - `stacks-blockchain:feat-fix-something`
+     - `stacks-blockchain:pr-112`
+   2. CI tests are run
+3. PR `develop` to the default branch where the PR is numbered `112`
+   1. Docker image tagged with the **branch name** and **PR number**
+   - ex:
+     - `stacks-blockchain:feat-fix-something`
+     - `stacks-blockchain:pr-112`
+   2. CI tests are run
 4. Merge `develop` branch to the default branch
-   1. CI Workflow is triggered, resulting in a pushed docker image tagged with the **default branch name**
+   1. Docker image is tagged with the **default branch** `master`
+   - ex:
+     - `stacks-blockchain:master`
+   2. CI tests are run
 5. CI workflow is manually triggered on **non-default branch** with a version, i.e. `2.1.0.0.0-rc0`
-   1. Github release for the manually input version is created with binaries
-   2. Docker image pushed with tags of the **input version** and **branch**
+   1. No Docker images/binaries are created
+   2. All release tests are run
 6. CI workflow is manually triggered on **default branch** with a version, i.e. `2.1.0.0.0`
    1. Github release for the manually input version is created with binaries
-   2. Docker image pushed with tags of the **input version** and **latest**
+   2. All release tests are run
+   3. Docker image pushed with tags of the **input version** and **latest**
+   - ex:
+   - `stacks-blockchain:2.1.0.0.0-debian`
+   - `stacks-blockchain:latest-debian`
+   - `stacks-blockchain:2.1.0.0.0`
+   - `stacks-blockchain:latest`
+
+## Tests
```

Comment:
Can you add some guidance on where we should put new integration tests?  Perhaps a table that lists out all the test batches, and what kinds of tests would need to go into them?

Reply:
This will take some time to document, but it's a good idea. i'll make it so.

Reply:
added to the `./docs/ci-release.md` file - i don't think i can answer *where* a dev may want to put a test, but i tried to explain why the current tests are are in each file. 
https://github.com/stacks-network/stacks-core/blob/ci/faster-tests/docs/ci-release.md?plain=1#L74-L136